yoti-sdk-spring-boot-example/src/main/resources/templates/index.html

```diff
@@ -1,22 +1,62 @@
 <!DOCTYPE html SYSTEM "http://www.thymeleaf.org/dtd/xhtml1-strict-thymeleaf-spring4-4.dtd">
 
-<html xmlns="http://www.w3.org/1999/xhtml"
-        xmlns:th="http://www.thymeleaf.org">
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org" class="yoti-html">
+
     <head>
-        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
-        <title>Home</title>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+        <title>Yoti client example</title>
+        <link rel="stylesheet" type="text/css" th:href="@{/static/index.css}" />
+        <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet" />
         <script src="https://sdk.yoti.com/clients/browser.2.2.0.js"></script>
     </head>
-    <body>
-    <p th:text="${rememberMeId}"/>
-    <!-- This span will create the Yoti button -->
-    <span data-th-attr="data-yoti-application-id=${applicationId}">
-        Use Yoti
-    </span>
-
-    <!-- This script snippet will also be required in your HTML body -->
-    <script>
-        _ybg.init()
-    </script>
+
+    <body class="yoti-body">
+
+        <main>
+            <section class="yoti-top-section">
+                <div class="yoti-logo-section">
+                    <img class="yoti-logo-image" th:src="@{/static/assets/logo.png}" th:srcset="@{/static/assets/logo@2x.png 2x}" alt="Yoti"/>
+                </div>
+
+                <h1 class="yoti-top-header">We now accept Yoti</h1>
```

Comment:
This wording is not the same as the original template which says 'Business Name now uses Yoti'.
Are we changing this?

Reply:
Well spotted.
Yes, we are changing it.  Agree with Kiran it's not worth making people configure a 'Business Name' in the app for the sake of displaying it here.  
Noted in SDK-493.